src/index.js

```diff
@@ -2,17 +2,19 @@ const testRunner = require('testrunner')
 const { configuration } = require('./configuration')
 const { generateReports } = require('./reporter')
 
-// run
-testRunner(configuration)
-  .then(async results => {
+async function run() {
+  try {
+    const results = await testRunner(configuration)
     // generate reports
     await generateReports({
-      types: ['json', 'markdown'],
       fileName: 'act-axe-result',
       outputDirectory: 'output',
-      results
+      fileContent: results
     })
-  })
-  .catch(error => {
-    throw new Error(error)
-  })
+  } catch (err) {
+    throw new Error(err)
```

Comment:
You don't need the try / catch statement here if you're just going to catch and re-throw the error.